packages/rum-core/src/domain/contexts/viewContexts.ts

```diff
@@ -30,6 +30,13 @@ export function startViewContexts(lifeCycle: LifeCycle): ViewContexts {
     viewContextHistory.closeActive(endClocks.relative)
   })
 
+  lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, (data: ViewEvent) => {
+    const currentView = viewContextHistory.find()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I don't think two view can be updated in parallel but just in case:
```suggestion
    const currentView = viewContextHistory.find(d(data.startClocks.relative))
```